dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/InQueryFilter.java

```diff
@@ -63,13 +65,16 @@ public InQueryFilter( String field, QueryFilter queryFilter )
      * @param encodedFilter actual "in" parameters
      * @return a SQL condition representing this InQueryFilter
      */
-    public String getSqlFilter( String encodedFilter )
+    public String getSqlFilter( String encodedFilter, boolean isText )
```

Comment:
Please add a description to explain the initial problem and why this fixes it :-)